### ISSUE TITLE
SCD Cloud + RAL Tier1 dashboards

### ISSUE THREAD

**Issue by tofu-rocketry** [apel/grafana-dashboards#41]
Depends on #40.

Once #40 is implemented, we can add SCD Cloud and RAL Tier1 dashboards (probably in separate folders, which will hopefully simplify permissions).

**Comment by tswinb** [comment]
RAL Tier-1 Dashboard requirements: one graph showing LHC Usage and one showing Non-LHC usage. Features such as allocation and capacity would be nice-to-haves.
